File: lib/layer2_core/dark/e_outline_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EOutlineDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd45;
      case 'standard': return cstd45;
      case 'blue': return cbd45;
      case 'green': return cgd45;
      case 'red': return crd45;
      case 'nothing': return cnod45;
      default: return cstd45;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EOutlineDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOutlineDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd45
            AppTheme.STANDARD -> R.color.cstd45
            AppTheme.BLUE -> R.color.cbd45
            AppTheme.GREEN -> R.color.cgd45
            AppTheme.RED -> R.color.crd45
            AppTheme.NOTHING -> R.color.cnod45
        }
    }
}
